Workbooks/SapMonitor2.0/AIOpsInsights/Availability Insights/Auto RCA.workbook

```diff
@@ -186,7 +186,97 @@
               "showDefault": false
             },
             "defaultValue": "value::all",
-            "queryType": 12
+            "queryType": 12,
+            "value": [
+              "value::all"
+            ]
+          },
+          {
+            "id": "5f818549-c01f-4a9c-b804-d7cd18e67d60",
+            "version": "KqlParameterItem/1.0",
+            "name": "SAPComponent",
+            "type": 1,
+            "isHiddenWhenLocked": true,
+            "value": "SAP Hana"
```

Comment:
Component name is `SAPComponent`
So wondering if this value will always be `SAP HANA`

Reply:
same as previous comments.